Cause LIBDIR and USRLIBDIR is set to the same path on alpine the symlink
can be relative.

--- a/Makefile
+++ b/Makefile
@@ -200,7 +200,7 @@ ifeq ($(NO_SOLIB),0)
 	$(INSTALL) -D $(LIBNAME) $(DESTDIR)$(LIBDIR)/$(LIBNAME)
 	$(LNS) $(LIBNAME) $(DESTDIR)$(LIBDIR)/$(SONAME)
 	mkdir -p $(DESTDIR)$(USRLIBDIR)
-	$(LNS) $(LIBDIR)/$(SONAME) $(DESTDIR)$(USRLIBDIR)/$(DEVELLIB)
+	$(LNS) $(SONAME) $(DESTDIR)$(USRLIBDIR)/$(DEVELLIB)
 	sed \
 	-e 's,@VERSION\@,$(VERSION),g' \
 	-e 's,@prefix\@,$(PREFIX),g' \
